frappe/core/doctype/user/user.py

```diff
@@ -983,4 +983,19 @@ def throttle_user_creation():
 	if frappe.flags.in_import:
 		return
 	if frappe.db.get_creation_count('User', 60) > 60:
-		frappe.throw(_('Throttled'))
\ No newline at end of file
+		frappe.throw(_('Throttled'))
+
+@frappe.whitelist()
+def update_roles(role_name):
```

Comment:
role_name or `role_profile_name` ??? Please setup correct naming!!!

This method should be called when saving User too ?